.github/workflows/package.yml

```diff
@@ -0,0 +1,31 @@
+name: package
+on:
+  release:
+    types: [published]
+jobs:
+  package:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ ubuntu-latest, windows-latest, macos-latest ]
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python
+        uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Install tox
+        run: |
+          pip install --upgrade pip
+          pip install tox
+      - name: Run packaging step
+        run: |
+          tox -e package
+      - name: Upload binary to release
+        run: |
+          gh release upload \
+            ${{ github.event.release.tag_name }} \
+            "$GITHUB_WORKSPACE/build/*/debug/install/isic*#${{ runner.os }} executable"
```

Comment:
@danlamanna any idea how to test this without actually generating a release? I won't be convinced that this shell invocation works until we actually run the `gh release upload` command...

Reply:
I'm just going to dump the files on an existing release as a test, I don't think it will do much harm.